dashboard/src/modules/components/SidebarComponent/index.jsx

```diff
@@ -12,27 +19,30 @@ const Menu = () => {
   const { pathname } = useLocation();
 
   const activeItem = useSelector((state) => state.sidebar.activeMenuItem);
+  const isLoggedIn = Cookies.get("isLoggedIn");
   const onSelect = (result) => {
     dispatch(setActiveItem(result.itemId));
   };
 
   useEffect(() => {
     if (pathname) {
-      const currPath = pathname.split("/").at(-1);
+      const modPathName = pathname.replace(/\/+$/, "");
+      const currPath = modPathName.split("/").at(-1);
+
       dispatch(setActiveItem(currPath));
     }
-  }, [dispatch, pathname]);
+  }, [dispatch, isLoggedIn, pathname]);
   return (
-    <Nav onSelect={onSelect}>
-      {menuOptions.map((item, index) => {
-        return (
-          <NavGroup
-            key={index}
-            aria-label={item.group.title}
-            title={item.group.title}
-          >
-            {item.submenu.map((option) => {
-              return (
+    <>
```

Comment:
Is this empty tag necessary?  (If so, can you help me understand why?)

It looks to me like the result of the ternary at line 37 is going to be a `<Nav>` tag either way, so I don't see what wrapping the expression in an empty tag accomplishes.

Given that, in both branches of that ternary, we start with `<Nav onSelect={onSelect}>`, perhaps you could use that here instead of the empty tag and have the ternary expression resolve to just the `<NavGroup>`'s or the `<NavList>`.

Reply:
Yes, the empty tag is necessary. Parsing error was thrown. 

Modified the expressions and empty is now omitted.